src/status_im/contexts/chat/messenger/messages/content/view.cljs

```diff
@@ -113,6 +115,34 @@
       constants/content-type-system-message-mutual-event-sent
       [system-message-contact-request message-data :contact-request])))
 
+(defn bridge-message-content
+  [{:keys [bridge-message timestamp]}]
+  (let [{:keys [user-avatar user-name
+                bridge-name content]} bridge-message
+        user-name                     (when (string? user-name)
```

Comment:
I think there is a conflict between the name of the destructed key `user-name` and the variable created on line 122. Perhaps a distinction between the two?

Reply:
no conflicts , it's intentional